tests/unit/IonTextWriterTest.js

```diff
@@ -311,6 +311,36 @@
       badWriterTest('Cannot end struct with missing field value',
           writer => { writer.stepIn(ion.IonTypes.STRUCT); writer.writeFieldName('foo'); writer.stepOut() });
 
+      //  annotations
+
+      badWriterTest('Cannot pass single string as an annotation.',
+          (writer) => { writer.setAnnotations('taco'), writer.writeInt(5) });
+      badWriterTest('Cannot add int as annotation.',
+          (writer) => { writer.addAnnotation(5), writer.writeInt(5) });
+      badWriterTest('Cannot add array of chars.',
+          (writer) => { writer.addAnnotation(['t', 'a', 'c', 'o']), writer.writeInt(5) });
+      badWriterTest('Cannot pass annotations array without a string.',
+          (writer) => { writer.setAnnotations([5]), writer.writeInt(5) });
+      badWriterTest('Cannot pass annotations array containing a non string value.',
+          (writer) => { writer.setAnnotations(['a', 5,'t']), writer.writeInt(5) });
+      badWriterTest('Cannot pass annotations array containing undefined.',
+          (writer) => { writer.setAnnotations([undefined]), writer.writeInt(5) });
+      badWriterTest('Cannot pass annotations array containing null',
+          (writer) => { writer.setAnnotations([null]), writer.writeInt(5) });
```

Comment:
add a happy case test for `setAnnotations(null)`

Reply:
done